WooCommerce/Classes/POS/Presentation/Payments Onboarding/PointOfSaleCardPresentPaymentOnboardingView.swift

```diff
@@ -0,0 +1,46 @@
+import SwiftUI
+
+/// Displays the WooPayments onboarding UI based on the state.
+struct PointOfSaleCardPresentPaymentOnboardingView: View {
+    @ObservedObject var viewModel: PointOfSaleCardPresentPaymentOnboardingViewModel
+
+    var body: some View {
+        VStack(spacing: Constants.verticalSpacing) {
+            HStack {
+                Spacer()
+                Button {
+                    viewModel.cancelOnboarding()
+                } label: {
+                    Image(systemName: "xmark")
+                        .font(.posButtonSymbol)
+                }
+                .accessibilityLabel(Localization.cancelOnboarding)
+                .foregroundColor(Color.posTertiaryText)
+            }
+            CardPresentPaymentsOnboardingView(viewModel: viewModel.onboardingViewModel)
+                // Hides the navigation bar title `navigationTitle` in `CardPresentPaymentsOnboardingView`.
+                .toolbar(.hidden)
+        }
+        .padding(Constants.padding)
+        .safariSheet(url: $viewModel.onboardingURL)
```

Comment:
❓  Question: Would it be worth to use `WebViewSheet` rather than `SafariSheetView` here? I haven't checked further but I think we can leverage the authenticated web view with the former.

Reply:
Good point - I followed the implementation in Menu > Payments to show the URL in a Safari sheet

https://github.com/woocommerce/woocommerce-ios/blob/5d92a93eaf73efa69b97dcf8fca3b15ac128570d/WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person%20Payments/Payments%20Menu/InPersonPaymentsMenuViewModel.swift#L238-L241

But I will do more testing when logging in with WPCOM and site credentials to see if it's possible to skip the login step when completing the WooPayments/Stripe form in the webview. Added a subtask: https://github.com/woocommerce/woocommerce-ios/issues/14233